modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/frontend/taglib/servlet/taglib/CPConfigurationListQualifiersScreenNavigationEntry.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.product.definitions.web.internal.frontend.taglib.servlet.taglib;
+
+import com.liferay.commerce.product.definitions.web.internal.display.context.CPConfigurationListQualifiersDisplayContext;
+import com.liferay.commerce.product.model.CPConfigurationList;
+import com.liferay.commerce.product.service.CPConfigurationEntryService;
+import com.liferay.commerce.product.service.CPConfigurationListRelService;
+import com.liferay.commerce.product.service.CPConfigurationListService;
+import com.liferay.commerce.product.service.CPDefinitionService;
+import com.liferay.commerce.product.service.CommerceCatalogService;
+import com.liferay.commerce.product.service.CommerceChannelRelService;
+import com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationEntry;
+import com.liferay.frontend.taglib.servlet.taglib.util.JSPRenderer;
+import com.liferay.portal.kernel.security.permission.resource.ModelResourcePermission;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.kernel.util.WebKeys;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Danny Situ
+ */
+@Component(
+	property = "screen.navigation.entry.order:Integer=10",
+	service = ScreenNavigationEntry.class
+)
+public class CPConfigurationListQualifiersScreenNavigationEntry
+	extends CPConfigurationListQualifiersScreenNavigationCategory
+	implements ScreenNavigationEntry<CPConfigurationList> {
+
+	@Override
+	public String getEntryKey() {
+		return getCategoryKey();
+	}
+
+	@Override
+	public void render(
```

Comment:
I've sent you a commit to make this entry visible only for list that are not master